### DISCUSSION TITLE
chd support

### DISCUSSION THREAD

**Discussion by telanus** [86Box/86Box#4253] [Ideas]
**Is your feature request related to a problem? Please describe.**
Add support for chd based compressed CD's and DVD's. this would allow the use of the mame CD Software list

**Describe the solution you'd like**
There is libchd(https://github.com/rtissera/libchdr) that can be used to support use of chd in 86Box. PPSSPP recently added chd to their list of supported formats, and their implementation can help: https://github.com/hrydgard/ppsspp/commit/ade64171ce40d1979d872e74f65ac4712f914c5c

**Comment by OBattler** [comment]
I would need a precompiler library I can use directly, not a library I have to compile myself - we would have to integrate that into the build process and keep it updated.

**Comment by strich** [comment]
Just to add some weight to this feature request: Adding CHD support is a really important piece of the puzzle for being able to scale out 86box for use in medium to large games archiving efforts.
Looking forward to any progress on this!

**Comment by treloret** [comment]
Dosbox-X has it integrated these days, and the major change over the past decade since CHD v5 was addition of zstd compression and .iso (dvd and cooked CDs) compression with createdvd, besides that it's mainly updates of backend libraries for the codecs (flac/lzma/zlib/zstd).

**Comment by snuq** [comment]
this would be amazing to add! CHD has halved the size of my playstation and ps2 libraries while keeping them fully functional, i would love to do the same with dos and windows cds/dvds
